fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/utility/RegionsConfigGammaDelayUtility.java

```diff
@@ -0,0 +1,124 @@
+        CRegion regionConfig = null;
+        try {
+            regionConfig = ConfigurationReader.importRegionConfig(regionConfigurationPath);
+        } catch (InternalFederateException e) {
+            throw new RuntimeException(e);
+        }
+
+        return regionConfig;
+    }
+
+    /**
+     * Sets all transmission delays (uplink, downlink.multicst, downlink.unicast) to 'GammaRandomDelay'.
+     *
+     * @param cRegion The region configuration object
+     * @param minDelay The minimum delay value (ns)
+     * @param expDelay The expected delay value (ns)
+     */
+    public static void setGammaRandomDelay(CRegion cRegion, long minDelay, long expDelay) {
+        GammaRandomDelay delay = new GammaRandomDelay();
+        delay.expDelay = expDelay;
+        delay.minDelay = minDelay;
+
+        for (CNetworkProperties region : cRegion.regions) {
+            region.downlink.multicast.delay = delay;
+            region.downlink.unicast.delay = delay;
+            region.uplink.delay = delay;
+        }
+    }
+
+    /**
+     * Writes the region configuration to a .json file.
+     *
+     * @param regionConfigurationPath The path to the output file (*.json)
+     * @param cRegion The region configuration object
+     */
+    public static void exportRegionsConfig(String regionConfigurationPath, CRegion cRegion) {
+        Gson gson = new GsonBuilder()
+                .registerTypeAdapter(GeoPoint.class, new GeoPolygonAdapter())
+                .setFieldNamingStrategy(new RegionsNamingStrategy())
+                .setPrettyPrinting().create();
+        try (Writer writer = new OutputStreamWriter(new FileOutputStream(regionConfigurationPath), Charsets.UTF_8)) {
+            gson.toJson(cRegion, writer);
+        } catch (IOException e) {
+            throw new RuntimeException("Error exporting regions config. \n"
+                    + "The file exists but is a directory rather than a regular file, does not exist but cannot be created, "
+                    + "or cannot be opened for any other reason. \n"
+                    + "Path: " + regionConfigurationPath);
+        }
+
+    }
+
+    public static void main(String[] args) {
+        if (args.length != 3) {
+            System.out.println("Usage: RegionsConfigUtility <path-to-regions.json> <minDelay (ns)> <expDelay (ns)>");
+        }
+
+        String regionConfigurationPath = args[0];
+        long minDelay = Long.parseLong(args[1]);
+        long expDelay = Long.parseLong(args[2]);
+
+        CRegion cRegion = getRegionsConfig(regionConfigurationPath);
+        setGammaRandomDelay(cRegion, minDelay, expDelay);
+        exportRegionsConfig(regionConfigurationPath, cRegion);
+    }
+
+
+    static class GeoPolygonAdapter extends TypeAdapter<GeoPoint> {
+        @Override
+        public void write(com.google.gson.stream.JsonWriter out, GeoPoint point) throws IOException {
+            out.beginObject();
```

Comment:
Do you have any idea why the RegionsNamingStrategy does not work and you have to do this instead?